waterlog/src/actions/SegmentLeaksByIdActions.js

```diff
@@ -1,29 +1,23 @@
-import {
-  FETCH_SEGMENTS_LEAKS_BEGIN,
-  FETCH_SEGMENTS_LEAKS_SUCCESS,
-  FETCH_SEGMENTS_LEAKS_FAILURE
-} from './Types';
-import { Globals } from './../Globals';
+import { FETCH_SEGMENTS_LEAKS_BEGIN, FETCH_SEGMENTS_LEAKS_SUCCESS, FETCH_SEGMENTS_LEAKS_FAILURE } from './Types';
```

Comment:
This was correct over multiple lines. 
please change it back

Reply:
Fixed